### ISSUE TITLE
The "ariaDescription" and "ariaDescribedBy" are not read by the screen reader

### ISSUE THREAD

**Issue by ionutprilipceanu** [nvaccess/nvda#17000]
### Steps to reproduce:

- Open NVDA Speech Viewer
- Go to < nav > element that have an aria-label="label" and aria-description="description" 
- Check NVDA 

or 

- Open NVDA Speech Viewer
- Go to < nav > element that have an aria-labelledby="id-labelledby" and aria-describedby="id-describedby"
- Check NVDA 

Code example below: 
< button >test< /button >
< nav aria-label="label" aria-description="description" >
	< a href="www.google.com">Google</ a >
</ nav	>
< input type="text" value="value" placeholder="placeholder" disabled>

Here is the link to codepen example: 
https://codepen.io/teodorachiosa/pen/VwNgPvE?editors=1100

### Actual behavior:
The "aria-description" and "aria-describedby" are not read by the screen reader.

### Expected behavior:
The "aria-description" and "aria-describedby" should be read by the screen reader.

**Comment by ionutprilipceanu** [comment]
Hi, here is the link to the Codepen
https://codepen.io/teodorachiosa/pen/VwNgPvE?editors=1100

**Comment by SaschaCowley** [comment]
Welcome @ionutprilipceanu ,

We are unable to process this issue as it stands.

- Please fill out the issue template fully.
- Many developers, contributors and community members here are vision impaired. Screenshots are not accessible. Provide an alternative description, and capture any relevant information from images.
- Please link the code sample in the initial comment documenting the issue.

Kind Regards,
NV Access Software Developers

**Comment by Adriani90** [comment]
@ionutprilipceanu are you using the mouse when navigating the website? Using the mouse, aria-description is not reported indeed but this is expected since visually abled users would see a hover popping up or a description showing up, they can focus it with the mouse when it appears.

Using the keyboard, here are my test results:
1. aria-Description is read when using arrow keys in browse mode, but not when using tab or shift+tab key in focus mode. This is expected because otherwise focus mode would get too verbose on using tab and shift+tab on elements. Focus mode is meant to focus on the labels of elements, not on their description.
2. aria-Label is reported when using tab and shift+tab or quick navigation key in browse mode, or when pressing nvda+shift+o to report the object focused by the navigator. NVDA reports both aria-label and the link text in these cases. Mostly, users use arrow keys in browse mode, quick nav key in browse mode or tab and shift+tab keys in focus mode. In this particular minimal test case, NVDA users are able to gather all information about this link, using the propper navigation patern. Lable, description and link text are read by NVDA.

Closing as works for me.

Please fill a separate issue with a minimal test case for aria-describedby, this is a different use case.
